rxjava2/src/main/java/io/vertx/reactivex/core/json/ObservableUnmarshaller.java

```diff
@@ -0,0 +1,78 @@
+package io.vertx.reactivex.core.json;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import io.reactivex.ObservableOperator;
+import io.reactivex.Observer;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.disposables.Disposable;
+import io.vertx.core.buffer.Buffer;
+
+import java.io.IOException;
+
+import static io.vertx.core.json.Json.mapper;
+import static java.util.Objects.nonNull;
+
+/**
+ * An operator to unmarshall json to pojos.
+ *
+ * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
+ */
+public class ObservableUnmarshaller<T, B> implements ObservableOperator<T, B> {
```

Comment:
I mean that there is no equivalent to `Observable.Transformer` in RxJava2 and the compose use the `ObservableTransformer` I cited above that is not much different of the `ObservableOperator` I used in the PR